### PR TITLE
MOSIP-37256 : Updated readme for developer guide java21

### PR DESCRIPTION
**Pull request by GOKULRAJ136**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,20 +11,21 @@ As the name suggests, Commons refers to all the common services (also called "ke
 Refer to [SQL scripts](db_scripts).
 
 ## Build & run (for developers)
-The project requires JDK 1.11. 
+The project requires JDK 21.0
```
Comment (dhanendra06) [inline README.md:14]: correct the j21 version

File: kernel/kernel-otpmanager-service/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Kernel Otp Manager Service
 
 ## Overview
-This service provides funtions to generate and validate OTP.
+This service provides functions to generate and validate OTP.
```
Comment (dhanendra06) [inline kernel/kernel-otpmanager-service/README.md:4]: put here also complete thing build and run.

**Review by dhanendra06 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (GOKULRAJ136) [reply README.md]: updated the correct java version

File: kernel/kernel-otpmanager-service/README.md (hunk 2 above)
Comment (GOKULRAJ136) [reply kernel/kernel-otpmanager-service/README.md:4]: added the build and run

**Review by dhanendra06 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**